orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/CompoundExecutionOperator.java

```diff
@@ -25,6 +25,7 @@
 import java.time.Duration;
 import java.util.function.Consumer;
 import java.util.function.Supplier;
+import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import lombok.NonNull;
```

Comment:
is `lombok.NonNull` unused now?

Reply:
missed a spot, but fixed it now, thanks!